desktop_version/src/Game.cpp

```diff
@@ -5907,6 +5917,50 @@ void Game::customloadquick(const std::string& savfile)
             map.roomnameset = true;
             map.roomname_special = true;
         }
+        else if (SDL_strcmp(pKey, "currentregion") == 0)
+        {
+            map.currentregion = help.Int(pText);
+        }
+#if !defined(NO_CUSTOM_LEVELS)
```

Comment:
You do not need a check for `NO_CUSTOM_LEVELS`:
1. This is already in the quicksave load function that is specifically for custom levels (`customloadquick`).
2. We actually removed `NO_CUSTOM_LEVELS` and `NO_EDITOR` builds a while ago. (To be fair, when you originally wrote this we still had them.)
```suggestion

```